features/storage/filesystem/littlefs/littlefs/emubd/lfs_emubd.c

```diff
@@ -105,7 +105,12 @@ int lfs_emubd_read(const struct lfs_config *cfg, lfs_block_t block,
         }
 
         size_t res = fread(data, 1, size, f);
-        if (res < size && !feof(f)) {
+        if (res < size
+#if !defined(__MICROLIB)
```

Comment:
I wonder if this `&& !feof(f)` test should ever be there.

Why would it be valid to attempt to read off the end of the file?

Reply:
I rever this file changes as build system will ignore this file due to .mbedignore